ctapipe/containers.py

```diff
@@ -922,3 +892,23 @@ class SimulatedShowerDistribution(Container):
         None, "array of core-distance bin lower edges, as in np.histogram", unit=u.m
     )
     histogram = Field(None, "array of histogram entries, size (n_bins_x, n_bins_y)")
+
+
+class DataContainer(Container):
```

Comment:
`ArrayEventContainer`?

Reply:
Sure, might as well rename it now.  I think at this point it really does only contain event-wise info (finally!).